platform-emf/src/main/java/com/softicar/platform/emf/table/configuration/EmfTableConfiguration.java

```diff
@@ -503,9 +505,27 @@ private void customizeAttributesAndOrder(EmfAttributeList<R, P> attributes) {
 
 	public void setBusinessKey(IDbKey<R> businessKey) {
 
+		checkBusinessKey(businessKey);
 		this.businessKey = businessKey;
 	}
 
+	private void checkBusinessKey(IDbKey<R> businessKey) {
+
+		if (!businessKey.isUniqueKey()) {
+			throw new SofticarDeveloperException(
+				"The set business key (%s) of table %s must be a unique key.",
```

Comment:
```suggestion
				"The business key (%s) of table %s must be a unique key.",
```
IMO, the word `set` is unnecessary and might confuse the reader